VideoLocker/res/values/errors.xml

```diff
@@ -63,6 +63,19 @@
     <!-- Alert dialog content shown when user attempts to view a video that hasn't been synced -->
     <string name="video_offline_message">Please select a video that you’ve downloaded onto your device.</string>
 
+    <!-- New version available -->
+    <!-- Snackbar notification shown when the current version of the app is unsupported -->
```

Comment:
@1zaman Please update strings for app_version_unsupported, app_version_deprecated, and app_version_outdated per the conversation [here](https://github.com/edx/edx-app-android/pull/748#issuecomment-226272349).